CHANGELOG.md

```diff
@@ -25,4 +25,3 @@
  - Plugins were updated to follow the new shutdown semantic, this mainly allows Logstash to instruct input plugins to terminate gracefully,
    instead of using Thread.raise on the plugins' threads. Ref: https://github.com/elastic/logstash/pull/3895
  - Dependency on logstash-core update to 2.0
-
```

Comment:
Unnecessary change

Reply:
Sorry. My editor do this automatically. Should I revert it?

Reply:
Yes please

Reply:
Fixed in 4248975